src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -91,35 +113,13 @@ export const TradeInput = ({ isCompact, tradeInputRef }: TradeInputProps) => {
     selectIsAnyAccountMetadataLoadedForChainId(state, isAnyAccountMetadataLoadedForChainIdFilter),
   )
 
-  const shouldShowTradeQuoteOrAwaitInput = useAppSelector(selectShouldShowTradeQuoteOrAwaitInput)
-  const isTradeQuoteRequestAborted = useAppSelector(selectIsTradeQuoteRequestAborted)
-  const hasUserEnteredAmount = useAppSelector(selectHasUserEnteredAmount)
-  const activeQuote = useAppSelector(selectActiveQuote)
-
-  const isSnapshotApiQueriesPending = useAppSelector(selectIsSnapshotApiQueriesPending)
-  const votingPower = useAppSelector(state => selectVotingPower(state, votingPowerParams))
+  const isKeplr = useMemo(() => !!wallet && isKeplrHDWallet(wallet), [wallet])
```

Comment:
Note for a follow-up: we can probably remove this, historical reasons were Osmosis swaps were quite borky in Keplr (or not implemented, can't recall exactly) iirc, but with swapper v2 and custom receive addies, this should now work (and will even better once we bring EVM support to Keplr)

Reply:
https://github.com/shapeshift/web/pull/2971 for historical ref